### ISSUE TITLE
Mixin error when loading questaddition mod

### ISSUE THREAD

**Issue by Sam-Chai** [IzzelAliz/Arclight#900]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.18.2-1.0.7-40114eb5

### OS & Java versions

OS: Windows 10, Java: jdk-17.0.1+12

### Plugins and Mods

```raw
cannot start server, so I cannot run /forge mods command.
Here is the mod list:
- architectury-4.10.88-forge.jar
- ftb-chunks-forge-1802.3.12-build.217.jar
- ftb-library-forge-1802.3.9-build.167.jar
- ftb-quests-forge-1802.3.13-build.175.jar
- ftb-teams-forge-1802.2.9-build.88.jar
- item-filters-forge-1802.2.8-build.47.jar
* questsadditions-1.18.2-1.4.1.jar (This mod error)
```

### Description

When I start my server, then crash me a Mixin error.
Here is the pastebin: [https://pastebin.com/sHCsrc4Y
](https://pastebin.com/sHCsrc4Y)

### Step to reproduce

1. Install questsadditions
2. Start server
3. Crash

### Logs

The log just printed arclight's logo: [https://pastebin.com/DQNZ0P5t](https://pastebin.com/DQNZ0P5t)

**Comment by heipiao233** [comment]
Full debug.log,pls

**Comment by Sam-Chai** [comment]
> 

Here it is: https://pastebin.com/jegr3kst

**Comment by IzzelAliz** [comment]
Fixed in bf99715e
